content/pubs/pubs.bib

```diff
@@ -108,7 +108,7 @@ @article{bagging2019
 @article{graspy2019,
   title={GraSPy: Graph Statistics in Python},
   author={Chung, Jaewon and Pedigo, Benjamin D. and Bridgeford, Eric W. and Varjavand, Bijan K. and Vogelstein, Joshua T.},
-  journal={arXiv},
+  journal={Journal of Machine Learning Research},
   year={2019},
   month={april},
```

Comment:
not the right month. maybe it's fine. @j1c @bdpedigo would know?

Reply:
Its not yet actually published tho

Reply:
@j1c jovo wanted me to update it anyway